pkgs/development/python-modules/torch/default.nix

```diff
@@ -9,14 +9,18 @@
 
   # Build inputs
   numactl,
-  CoreServices, libobjc,
+  Accelerate, CoreServices, libobjc,
 
   # Propagated build inputs
   numpy, pyyaml, cffi, click, typing-extensions,
 
   # Unit tests
   hypothesis, psutil,
 
+  # Disable MKLDNN on aarch64-darwin, it negatively impacts performance,
+  # this is also what official pytorch build does
+  mklDnnSupport ? !(stdenv.isDarwin && stdenv.isAarch64),
```

Comment:
Actually, since you've verified that mklDnnSupport=false works better on x86 too, it would make sense to restore the original isDarwin condition. Other than that, LGTM

Reply:
mklDnnSupport=true works better on x86_64-darwin.

Reply:
Silly me